### PR TITLE
resolves doc issues 

### LINKED ISSUE #117: Add an example using custom filters in v-select
eg vuetifyjs/vuetify#2726

Maybe also searching multiple fields instead of just `text`

### PR DESCRIPTION
**Pull request by MustafaHussaini**
resolves #117 , #153, https://github.com/vuetifyjs/docs/issues/329, https://github.com/vuetifyjs/docs/issues/300,
https://github.com/vuetifyjs/docs/issues/293

### CODE REVIEW

**Comment by nekosaur** [comment]
Thank you for your contributions, but in the future I would prefer it if you create separate PRs for each issue, unless they are closely related.

**Comment by MustafaHussaini** [comment]
You are welcome, 
I will do

File: components/helpers/Parameters.vue [hunk 1]
```diff
@@ -59,7 +59,8 @@
               newItem[key] = item[key] == null ? '' : item[key]
             }
           }
-
+          if(item.textToDisplay)
+             newItem.name= this.genName(item.textToDisplay)
```
Comment (johnleider) [inline components/helpers/Parameters.vue]: is there a way we can get around this? I don't like conditionals like this

Comment (MustafaHussaini) [reply components/helpers/Parameters.vue]: i am trying to replace `newItem.name` with `item.textToDisplay` if it `item` has `textToDisplay` filed.
I did this for  props like `grid-list-{xs through xl}` 
i don't know any shorter solution.

**Comment by nekosaur** [comment]
You shouldn't manually edit `api/api.js`. It gets generated by a tool, so any changes you make will get overriden next time we generate it.

**Comment by nekosaur** [comment]
@MustafaHussaini I've submitted #168 which takes care of the grid component props, so please remove those changes from this PR.

**Comment by MustafaHussaini** [comment]
I am closing these PR for now 
and opening new PR with out changes resolved in #168

**Comment by nekosaur** [comment]
Please take this opportunity to create a separate PR for each issue :+1:

**Pull request closed without merging**